linux_pipeline/Jenkinsfile_lis_rpm_hyperv

```diff
@@ -29,28 +29,31 @@ properties ([
 ])
 }
 
+// defines deploy scenarios coverage - platform independent
```

Comment:
this is platform dependent, not independent?

Reply:
nodesMap is a generic definition, it is independent on the platforms as that is further done in validationNodesMap

Reply:
if it is platform independent, the nodesMap  should not be there